### PR TITLE
add in web_accessible_resources

### PR DESCRIPTION
**Pull request by andymckay**
* lets do web_accessible_resources as well

### CODE REVIEW

File: tests/test.web_accessible_resources.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import validate from '../validator';
+import { validManifest } from './helpers';
+import cloneDeep from 'lodash.clonedeep';
```
Comment (tofumatt) [inline tests/test.web_accessible_resources.js]: Not sure we have a rule about this, but similar to Python I like (and I believe we're been) loading third party libraries at the top, then an empty line, then our own libraries. Helps developers find where they should be looking for code. 😀

**Comment by tofumatt** [comment]
r+wc

File: tests/test.web_accessible_resources.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+describe('/web_accessible_resources', () => {
+
+  it('should be an array', () => {
+    var manifest = cloneDeep(validManifest);
+    manifest.web_accessible_resources = 'foo.png';
+    validate(manifest);
+    assert.equal(validate.errors.length, 1);
+    assert.equal(validate.errors[0].dataPath, '/web_accessible_resources');
```
Comment (EnTeQuAk) [inline tests/test.web_accessible_resources.js]: I would really love to see the actual error message being validated to match the expected error to make sure it's not masked by some other error message.

File: tests/test.web_accessible_resources.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  it('should fail if not an array of strings', () => {
+    var manifest = cloneDeep(validManifest);
+    manifest.web_accessible_resources = 1;
```
Comment (EnTeQuAk) [inline tests/test.web_accessible_resources.js]: Guess `["tabs", 1]` would be a better test? This test just tests exactly what `it('should be an array')` already tests.

**Comment by EnTeQuAk** [comment]
r+wc `#2`

File: tests/test.web_accessible_resources.js (hunk 1 above)
Comment (andymckay) [inline tests/test.web_accessible_resources.js]: Bah. I'm -1 on rules that need to be thought about and can't be enforced through automation 😞

File: tests/test.web_accessible_resources.js (hunk 2 above)
Comment (andymckay) [inline tests/test.web_accessible_resources.js]: Sure, aded.

File: tests/test.web_accessible_resources.js (hunk 3 above)
Comment (andymckay) [inline tests/test.web_accessible_resources.js]: That's a better test, good call.

**Pull request merged**